### ISSUE TITLE
I can't upload a video for a new sign

### ISSUE THREAD

**Issue by rosestamp** [Signbank/Global-signbank#768]
When I try to upload a video, I receive the following message
<img width="1792" alt="Screenshot 2021-09-01 at 14 45 01" src="https://user-images.githubusercontent.com/63103864/131682956-913663cf-8ed9-4527-98b6-29f3fcbc3e31.png">

**Comment by susanodd** [comment]
This might be related to #766 ? Perhaps the python files are not on the repaired server.

**Comment by rosestamp** [comment]
Hi, thanks for your message but I really don't understand what this means. Who can I ask about the python files? When will the issue be resolved? I have 1000 files to upload and I can't progress if I can't upload the videos

**Comment by susanodd** [comment]
Okay, that is a problem!

The python code files need to be on the server for it to be able to execute the required command. The code that resizes the video is in a different code repository. It looks like that code isn't on the new server. It basically needs to be copied.

We can see if @vanlummelhuizen can put the code on the new server (he authored the code)? Otherwise @ocrasborn needs to ask @Woseseltops.to check this. Normally, they would be getting alerts when github issues reference them. That's why I am putting them in the text. @ocrasborn has a direct link to @Woseseltops in case of major glitches. This actually sounds like it is.

**Comment by ocrasborn** [comment]
@Woseseltops , is this because we're still running a temporary installation following the server update of a few weeks ago? Did you run all the tests for that installation? See also #769 , same question for that issue

**Comment by Woseseltops** [comment]
> The python code files need to be on the server for it to be able to execute the required command. The code that resizes the video is in a different code repository. It looks like that code isn't on the new server. It basically needs to be copied.

Returned from paternity leave today, and copied the files over. Lost an hour before I figured out the permissions need to be different, but it's working now!

> @Woseseltops , is this because we're still running a temporary installation following the server update of a few weeks ago?

Yes, but it is not temporary as far as I'm concerned. I've wanted to move to Lightning anyway, as we have more freedom there (server admin wise)... a problem like the one we had on Applejack (being forced to upgrade while Signbank was not ready) will never occur on Lightning.

If we ever want to go back to Applejack, we would first need to upgrade our Django and Python versions.

> Did you run all the tests for that installation? 

Didn't get there yet (why check whether all details are working when you can already see major things that are not working with the naked eye :P), will do!

**Comment by rosestamp** [comment]
Hi, just to let you know that I still cannot upload videos despite the comment being closed. I don't receive the same error message but I click upload video and then no video appears. The site just shows 'no file chosen' as if I didn't do anything.

**Comment by susanodd** [comment]
I just tested this on the live server. It worked for me.
In Edit mode,
Directly above the blue "Upload Video"  button, there is grey "Browse" you need to use to open a window to select a file.

Can you try reloading the page to make sure it gets the newest code?

**Comment by susanodd** [comment]
It's Bladeren in the screenshot below, right above the blue Upload Video.

![bladeren_button](https://user-images.githubusercontent.com/24272544/133816878-c17d912c-b3f1-427f-8dd4-efed602b6c94.png)

**Comment by rosestamp** [comment]
Hi,
I tried again and it still didn't work. I have uploaded videos in the past with no problem. I pressed the Choose file button and I searched for a file. I selected it and it appears named next to the Choose file button and then I press upload and the page kind of refreshes and there is a blank video which doesn't play. It isn't clear if it is there or not. See attache
<img width="1772" alt="Screenshot 2021-09-17 at 17 45 39" src="https://user-images.githubusercontent.com/63103864/133824656-eeaa57cd-6e6c-45bf-b693-bfd4f86968e1.png">
d.
